consensus/api/proto/consensus_common.proto

```diff
@@ -97,6 +97,7 @@ enum ProposeTxResult {
     InputRuleInvalidAmountSharedSecret = 51;
     InputRuleTxOutConversion = 52;
     InputRuleAmount = 53;
+    LedgerTxOutIndexOutOfBounds = 54;
```

Comment:
This is the new status code that indicates the condition that we want clients to be able to retry on (without retrying all Ledger errors, many of which may not be recoverable)